src/main/java/frc/robot/Robot.java

```diff
@@ -20,6 +20,13 @@
  */
 public class Robot extends TimedRobot {
   public static final boolean isCompetition = true;
+  public static final boolean autonSubsytem = true;
+  public static final boolean batterySubsystem = true;
+  public static final boolean cameraSubsystem = true;
+  public static final boolean everybotIntakeSubsystem = true;
+  public static final boolean intakeSubsystem = true;
+  public static final boolean simulationSubsystem = true;
+  public static final boolean Swerve = true;
```

Comment:
Let's make this swerveSubsystem, please

Reply:
Changed name to Enabled instead of Subsystem